app/Http/Controllers/API/DeleteAccountController.php

```diff
@@ -0,0 +1,323 @@
+                                ])->delete();
+                            }
+
+                            $photo->delete();
+                        }
+                        catch (\Exception $e)
+                        {
+                            \Log::info(['DeleteAccountController', $e->getMessage()]);
+
+                            return [
+                                'success' => false,
+                                'msg' => 'problem deleting photo'
+                            ];
+                        }
+                    }
+                }
+            });
+
+        try
+        {
+            $adminVerificationLogs = AdminVerificationLog::where('user_id', $userId)->count();
+
+            if ($adminVerificationLogs > 0)
+            {
+                AdminVerificationLog::where('user_id', $userId)->delete();
+            }
+
+            $cleanupUser = DB::table('cleanup_user')->where('user_id', $userId)->count();
+
+            if ($cleanupUser > 0)
+            {
+                DB::table('cleanup_user')->where('user_id', $userId)->delete();
+            }
+
+            $cleanups = Cleanup::where('user_id', $userId)->count();
+
+            if ($cleanups > 0)
+            {
+                Cleanup::where('user_id', $userId)->delete();
+            }
+
+            $countries = Country::select('id', 'created_by', 'user_id_last_uploaded')
+                ->where('created_by', $userId)
+                ->orWhere('user_id_last_uploaded', $userId)
+                ->get();
+
+            if (sizeof($countries) > 0)
+            {
+                foreach ($countries as $country)
+                {
+                    $country->created_by = null;
+                    $country->user_id_last_uploaded = null;
+                    $country->save();
+                }
+            }
+
+            $states = State::select('id', 'created_by', 'user_id_last_uploaded')
+                ->where('created_by', $userId)
+                ->orWhere('user_id_last_uploaded', $userId)
+                ->get();
+
+            if (sizeof($states) > 0)
+            {
+                foreach ($states as $state)
+                {
+                    $state->created_by = null;
+                    $state->user_id_last_uploaded = null;
+                    $state->save();
+                }
+            }
+
+            $cities = City::select('id', 'created_by', 'user_id_last_uploaded')
+                ->where('created_by', $userId)
+                ->orWhere('user_id_last_uploaded', $userId)
+                ->get();
+
+            if (sizeof($cities) > 0)
+            {
+                foreach ($cities as $city)
+                {
+                    $city->created_by = null;
+                    $city->user_id_last_uploaded = null;
+                    $city->save();
+                }
+            }
+
+            $littercoin = Littercoin::where('user_id', $userId)->get();
```

Comment:
This can be just `Littercoin::where('user_id', $userId)->delete();`.
You can do similar simplifications to the other parts below.